docs/cpan-starjacking.md

```diff
@@ -0,0 +1,120 @@
+Some other indicators can help in the choice but give more informations about *quality*:
+- Issues
+![Issues](../media/starjacking/issues.png)
+- CPANTesters
+![CPAN Testers](../media/starjacking/testers.png)
+- Bus factor
+![Bus factor](../media/starjacking/bus.png)
+
+## StarJacking ![Stars](../media/starjacking/stars.png)
+StarJacking refers to the trust that VCS stars provide to a module.
+
+By reusing the popularity of another project, unrelated to the module,
+and exploiting the non-verification of the link (metadata),
+an attacker can make a module appear much more popular than it actually is.
+
+Other ecosystems like PyPI and npm are also impacted by this problem.
+
+Like many other programming language ecosystems, we can attach "metadata"
+such as a public repository, website and issue tracker URLs to CPAN packages.
+
+It has to be done directly in the release metadata.
+
+Example of using resources metadata pointing to a popular project, for instance in `META.yml`
+```
+META_MERGE   => {
+    resources => {
+        repository => {
+            type => 'git',
+            url  => 'https://github.com/popular/lot-of-stars.git',
+            web  => 'https://github.com/popular/lot-of-stars',
+        },
+        bugtracker => {
+            web => 'https://github.com/popular/lot-of-stars-and-tickets',
+        },
+    },
+},
+```
+
+Read [CPAN add link to version control system](https://perlmaven.com/how-to-add-link-to-version-control-system-of-a-cpan-distributions) for more about how to set the metadata.
+
+CPAN metadats are effectively unverified.
+
+PyPI now implements a mechanism for verifying metadata with [Project Metadata Verifications](https://docs.pypi.org/project_metadata/#verified-details):
+
+![PyPI verified](../media/starjacking/pypi-verified.png)
+
+Or it can warn you about metadata information being "unverified":
+
+![PyPI unverified](../media/starjacking/pypi-unverified.png)
+
+Back to MetaCPAN, the outcome is that GitHub/Gitlab/... numbers are almost invisible in MetaCPAN. For instance, for a module with more than 200 stars in GitHub:
+
+![GitHub numbers in MetaCPAN](../media/starjacking/metacpan-side.png)
+
+Only issues are visible, but how can issues help in StarJacking?
+
+Still, we can get more details on metadata by hovering **Repository**:
+
+![GitHub Metadata in CPAN](../media/starjacking/github.png)
+
+As a conclusion, CPAN is not impacted even though CPAN modules can provide inaccurate or incorrect metadata.
+
+It's because GitHub's popularity is not strongly enough propagated to MetaCPAN, so lying on metadata (in a particular repository) does not bring any real advantage over other modules.
+
+## PlusPlusJacking ![PlusPlus](../media/starjacking/plusplus.png)
+MetaCPAN package popularity uses an independant indicator, the "++" (PlusPlus).
+
+++ are sent by other CPAN authors.
+
+Manipulation of a PlusPlus score is possible but not cheap: it require creation of several spammy authors.
```

Comment:
Can we find a better (more precise) term to use than _spammy authors_?

Reply:
cheap here was used in the surrounding context of it being hard to create several new accounts on PAUSE. Perhaps rewording to something like _"Manipulation of a PlusPlus score is arguably possible but not straightforward, as it requires the registration of several new accounts, which is a human-reviewed process."_